mne/viz/epochs.py

```diff
@@ -844,6 +844,9 @@ def _prepare_mne_browse_epochs(params, projs, n_channels, n_epochs, scalings,
     # Draw event lines for the first time.
     _plot_vert_lines(params)
 
+    # default key to close window
+    params['close_key'] = 'escape'
```

Comment:
I'm not sure I'd change this for the annotation window - at least the behavior of the escape key is documented there. Which selection dialog do you mean? The projections dialog? I wouldn't change it there either, because this sub-dialog is similar to the annotations dialog (and I never show the projections button in MNELAB anyways).